lib/dom.js

```diff
@@ -17,6 +17,9 @@ declare class Blob {
   type: string;
   close(): void;
   slice(start?: number, end?: number, contentType?: string): Blob;
+  arrayBuffer(): Promise<Arraybuffer>;
```

Comment:
Looks like capitalization is wrong -- it should be `ArrayBuffer` with a capital `B`, right?